### PR TITLE
CI: Upgrade actions

### PR DESCRIPTION
**Pull request by Margen67**
Split from #2470
Closes #2463

### CODE REVIEW

**Comment by RadWolfie** [comment]
autoclose.yml can be update yet will need auto-close-issues action update to node20 first.

**Comment by RadWolfie** [comment]
Update: auto-close-issues has been updated, can you update autoclose.yml file as well?

```diff
-        uses: actions/checkout@v3
+        uses: actions/checkout@v4
...
-        uses: ergo720/auto-close-issues@v1.0.4
+        uses: ergo720/auto-close-issues@v1
```

**Comment by Margen67** [comment]
> UPDATE: auto-close-issues has been updated, can you update autoclose.yml file as well?
> 
> ```diff
> -        uses: actions/checkout@v3
> +        uses: actions/checkout@v4
> ...
> -        uses: ergo720/auto-close-issues@v1.0.4
> +        uses: ergo720/auto-close-issues@v1
> ```

Done.

File: .github/workflows/pull-request.yml [hunk 1]
```diff
@@ -7,7 +7,7 @@ jobs:
     runs-on: ubuntu-latest
     steps:
       - name: Labeler
-        uses: actions/labeler@v4
+        uses: actions/labeler@v5
```
Comment (RadWolfie) [inline .github/workflows/pull-request.yml:10]: If we are to upgrade this labeler action, then `.github/labeler.yml` file will need an update as well. v5 isn't compatible with v4's labeler.yml configuration.

reference:
- https://github.com/Cxbx-Reloaded/XbSymbolDatabase/commit/1cbb8e32ad4694cbca9714499812c99146268962 (`.github/labeler.yml` changes)
- https://github.com/actions/labeler/tree/main?tab=readme-ov-file#breaking-changes-in-v5 (order list number 1)

Comment (Margen67) [reply .github/workflows/pull-request.yml:10]: Did I do it right?

**Review by RadWolfie — APPROVED** [review]

**Pull request merged**